src/nodes.coffee

```diff
@@ -5348,34 +5353,61 @@ unfoldSoak = (o, parent, name) ->
   ifn
 
 # Constructs a string or regex by escaping certain characters.
-makeDelimitedLiteral = (body, options = {}) ->
-  body = '(?:)' if body is '' and options.delimiter is '/'
+makeDelimitedLiteral = (body, {delimiter: delimiterOption, escapeNewlines, double, includeDelimiters = yes, escapeDelimiter = yes} = {}) ->
+  body = '(?:)' if body is '' and delimiterOption is '/'
+  escapeTemplateLiteralCurlies = delimiterOption is '`'
   regex = ///
       (\\\\)                               # Escaped backslash.
     | (\\0(?=[1-7]))                       # Null character mistaken as octal escape.
```

Comment:
As far as I remember, this code is here to make sure that `"\0#{}7"` compiles to `` `\x007` ``. But id didn't seem to work in [try CoffeeScript](http://coffeescript.org/#try:%22%5C0%23%7B%7D7%22).

Either way, this should probably deal with `\08`, `\08` and `\09` as well, since they are invalid in strict mode and template literals.